content/guides/debugging/_index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Incident Debugging Workflow"
+date: 2018-11-26T22:22:31-07:00
+draft: false
+aliases: [/integrations/workflow]
+weight: 1
```

Comment:
Why are we changing this weight? I'd expect this to be 3 given the ordering currently: HTTP, gRPC, Debugging

Reply:
OK. gRPC was at 70, so I changed this to 80. I did not want to split gRPC and HTTP because they are more obviously related and both have sub-menus. Please see new commit.